frontend/src/components/CustomTable.tsx

```diff
@@ -406,6 +458,35 @@ export default class CustomTable extends React.Component<CustomTableProps, Custo
     return result;
   }
 
+  public handleChange = (name: string) => (event: any) => {
```

Comment:
Can we rename to something more specific if this only handles changes in the filter box?

Reply:
Sure. Renamed to `handleFilterChange`